src/components/order-service/order-service-review.html

```diff
@@ -27,33 +31,41 @@ <h3 class="review-message">
       <span class="sr-only">Success</span>
-      <h3 class="review-message">
-        <span>
-          <strong>{{$ctrl.serviceInstance.metadata.name}}</strong> has been added to <strong>{{$ctrl.projectDisplayName}}</strong> successfully
-        </span>
+      <h3 class="results-message">
+        <strong>{{$ctrl.serviceInstance.metadata.name}}</strong> has been added to <strong>{{$ctrl.projectDisplayName}}</strong> successfully.
       </h3>
     </div>
   </div>
   <div ng-if="$ctrl.orderComplete && $ctrl.binding">
     <bind-results error="$ctrl.bindError"
-                  progress-inline="true"
                   binding="$ctrl.binding"
                   secret-href="$ctrl.selectedProject | secretUrl : $ctrl.baseProjectUrl : $ctrl.binding.spec.secretName"
                   service-to-bind="$ctrl.serviceInstance.metadata.name"
                   bind-type="{{$ctrl.bindType}}"
                   application-to-bind="$ctrl.appToBind.metadata.name"
                   show-pod-presets="$ctrl.showPodPresets">
     </bind-results>
+    <p><a href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}">Continue to the project overview</a>.</p>
   </div>
-  <div class="alert alert-info" ng-if="$ctrl.orderComplete && $ctrl.bindType === 'none'">
-    <span class="pficon pficon-info" aria-hidden="true"></span>
-    <span class="sr-only">Info</span>
-    Continue to your project to bind this service to your application. Binding this service creates a secret containing the information necessary for your application to use the service.
+  <div ng-if="$ctrl.orderComplete && $ctrl.bindType === 'none'">
+    <p><a href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}">Continue to the project overview</a> to bind this service to your application. Binding this service creates a secret containing the information necessary for your application to use the service.</p>
   </div>
-  <div class="footer-panel">
-    <a class="btn btn-primary" href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}">View Project</a>
+  <div ng-if="!$ctrl.error">
+    <p class="or">or</p>
+    <p>Browse resources for {{$ctrl.serviceClass.name}}:</p>
+    <ul ng-if="$ctrl.serviceClass.resource.externalMetadata.documentationUrl || $ctrl.serviceClass.resource.externalMetadata.supportUrl || $ctrl.serviceInstance.status.dashboardURL">
+      <li ng-if="$ctrl.serviceClass.resource.externalMetadata.documentationUrl">
+        <a ng-href="{{$ctrl.serviceClass.resource.externalMetadata.documentationUrl}}" target="_blank">Documentation</a>
+      </li>
+      <li ng-if="$ctrl.serviceClass.resource.externalMetadata.supportUrl">
+        <a ng-href="{{$ctrl.serviceClass.resource.externalMetadata.supportUrl}}" target="_blank">Support</a>
+      </li>
+      <li ng-if="$ctrl.serviceInstance.status.dashboardURL">
+        <a ng-href="{{$ctrl.serviceInstance.status.dashboardURL}}">Service Dashboard</a>
```

Comment:
We should probably update the overview service instance row to use "Dashboard" instead of "Console" to be consistent.

Reply:
`target="_blank"`?